tests/safe_eval_test.py

```diff
@@ -76,6 +76,9 @@ def test_safe_eval4():
   assert safe_eval("myadd2(a= 39)", globals(), b=3) == -42
   assert safe_eval("myadd2(a= 39, b = 3)", globals()) == -42
 
+def test_safe_eval5():
+  assert safe_eval("myadd", globals()) == 42
+  assert safe_eval("myadd()", globals()) == 42
 
```

Comment:
you might need to add a test case like you mentioned in #10, like in the model layer, use binary or ternary for quantizers.

thanks.

Reply:
hi @vloncar  does my comment make sense?

Reply:
I made the change so that the feature is used in `qconvolutional_test.py`. Is that what you had in mind?

Reply:
yes! thanks.